pkg/client/package.go

```diff
@@ -51,10 +51,16 @@ func (c *packageClient) GetAll(ctx context.Context, result *v1alpha1.PackageList
 		Do(ctx).Into(result)
 }
 
-func (c *packageClient) Delete(ctx context.Context, pkg *v1alpha1.Package) error {
+func (c *packageClient) Delete(ctx context.Context, pkg *v1alpha1.Package, blocking bool) error {
```

Comment:
Not sure about the flag argument here, I would prefer two separate methods or to pass the propagation as parameter (since the client is a bit more "low-level" I could imagine it makes sense to expose this detail to the caller).

Reply:
Yes that's a good idea.